### PR TITLE
refactor(api,ui): delete useless type in pipeline

### PR DESCRIPTION
**Pull request by bnjjj**
Signed-off-by: Benjamin Coenen <benjamin.coenen@corp.ovh.com>

1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report ut-engine#8417.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report ut-engine#8418.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report ut-engine#8419.0 ✘  
* Stage 1
  * Engine Test with DB ✘

File: ui/src/app/model/pipeline.model.ts [hunk 1]
```diff
@@ -17,27 +17,27 @@ export class PipelineStatus {
     static STOPPED = 'Stopped';
 
     static neverRun(status: string) {
-      if (status === this.SKIPPED || status === this.NEVER_BUILT || status === this.SKIPPED || status === this.DISABLED) {
-        return true;
-      }
+        if (status === this.SKIPPED || status === this.NEVER_BUILT || status === this.SKIPPED || status === this.DISABLED) {
```
Comment (richardlt) [inline ui/src/app/model/pipeline.model.ts]: return directly

File: ui/src/app/model/pipeline.model.ts [hunk 2]
```diff
@@ -17,27 +17,27 @@ export class PipelineStatus {
...
 
-      return false;
+        return false;
     }
 
     static isActive(status: string) {
-      if (status === this.WAITING || status === this.BUILDING) {
-        return true;
-      }
+        if (status === this.WAITING || status === this.BUILDING) {
```
Comment (richardlt) [inline ui/src/app/model/pipeline.model.ts]: same

File: ui/src/app/model/pipeline.model.ts [hunk 3]
```diff
@@ -17,27 +17,27 @@ export class PipelineStatus {
...
 
-      return false;
+        return false;
     }
 
     static isDone(status: string) {
-      if (status === this.SUCCESS || status === this.STOPPED || status === this.FAIL) {
-        return true;
-      }
+        if (status === this.SUCCESS || status === this.STOPPED || status === this.FAIL) {
```
Comment (richardlt) [inline ui/src/app/model/pipeline.model.ts]: same

File: ui/src/app/model/pipeline.model.ts [hunk 4]
```diff
@@ -78,14 +77,14 @@ export class Pipeline {
...
-
-      if (!pipelineNamePattern.test(name)) {
-          return false;
-      }
-      return true;
+        if (!name) {
+            return false;
+        }
+
+        if (!pipelineNamePattern.test(name)) {
```
Comment (richardlt) [inline ui/src/app/model/pipeline.model.ts]: return directly

**Pull request merged**